model-optimizer/mo/utils/versions_checker.py

```diff
@@ -191,6 +216,7 @@ def get_environment_setup():
         exec("del tensorflow")
     except (AttributeError, ImportError):
         pass
+    env_setup['sys_platform'] = sys.platform
```

Comment:
Do we need to pre-processing for `sys.platform` value because for Linux systems it can vary from `linux` to `linux2`, etc. Please take a look at: https://docs.python.org/3/library/sys.html

Reply:
I checked this behavior and I think that we don't need pre-process for `sys.platform` value for Linux systems because `sys.platform` value doesn’t contain the major version since python 3.3 version and minimum supported python version for MO is 3.4.

Also according to docs we don't need to check existence of `platform` in `sys` module.